ethicalads-theme/templates/ea/publishers-calculator.html

```diff
@@ -0,0 +1,112 @@
+<section class="container py-5 mb-10">
+  <article class="page">
+    <h1 class="font-weight-light display-3">{{ page.title }}</h1>
+
+    <p>There's a few important variables that determine how much you'll make with EthicalAds. We built this calculator so you know what kind of a monthly payout to expect.</p>
+
+<div id="publisher-calculator" class="card card-border border-primary shadow-light-lg">
+  <div class="card-body">
+    <div class="row justify-content-center">
+      <div class="col-md-8 col-xs-12">
+
+        <form>
+
+          <div class="form-group mb-8">
+            <h3>Monthly site pageviews</h3>
+            <p class="small">How many pageviews your site gets is the most important input into how much you'll earn with us. Use "unique pageviews" if that is broken out in your analytics.</p>
+
+            <div class="d-print-none">
+              <div class="input-group input-group-sm mx-auto w-50 w-lg-25">
+                <input type="number" class="form-control" data-bind="value: pageviews" min="20000" max="5000000" step="10000" name="pageviews-precise" id="pageviews-precise">
+              </div>
+
+              <input type="range" class="form-control-range" min="20000" max="5000000" step="10000" data-bind="value: pageviews" name="pageviews" id="pageviews">
+              <div class="d-flex">
+                <div class="small flex-fill">20K</div>
+                <div class="small flex-fill text-right">5M</div>
+              </div>
+            </div>
+          </div>
+
+          <div class="form-group mb-8">
+            <h3>Pageview estimate source</h3>
+            <p class="small">Where did you get the figure for your monthly pageviews? Different ways of counting pageviews more or less accurately reflect how many pageviews will result in a billable ad impression (see <a href="/publishers/faq/#analytics-shows-more-pageviews-than-youre-showing-ad-views-why">why that is</a>).</p>
+
+            <div class="d-print-none">
+
+              <select class="form-control" data-bind="foreach: PAGEVIEW_SOURCES, value: pageview_source" name="pageview_source" id="pageview_source">
+                <option data-bind="text: text, value: value">-- select pageviews source --</option>
+              </select>
+            </div>
+          </div>
+
+          <div class="form-group mb-8">
+            <h3>Estimated CPM</h3>
+            <p class="small">Most publishers on our network earn between $2.25 and $2.75 CPM. However, more precisely you earn 70% of what <a href="/advertisers/#pricing">we charge advertisers</a> which varies based on the topic of your site and the geographic make-up of your audience. Sites where the audience is more heavily weighted toward North America and Western Europe or where the topic is in high demand like security, data science, or DevOps will be on the higher end of the range. In addition, some campaigns are sold on a per-click basis so a higher CTR will also increase your effective CPM.</p>
```

Comment:
This feels like way too much text.. Do we not want to ask users about their split in geos, with some reasonable defaults? That seems way cleaner than this copy and asking them to know what CPM they'll get. 

If we want to ship this version ASAP, some shorter copy here: 
 
```suggestion
            <p class="small">Most publishers on our network earn between $2.25 and $2.75 CPM. You earn 70% of what <a href="/advertisers/#pricing">we charge advertisers</a> which varies based on site topic and the geography of your audience. Our standard CPM numbers assume more than half of your audience is located in North America and Western Europe.</p>
```

Reply:
I don't think asking for a breakdown helps the calculator function too much. It would help accuracy but I don't think people know that breakdown so they'll probably just stick with the defaults. I say we just take shorter copy.